lib/kanal/plugins/user_system/user_system_plugin.rb

```diff
@@ -12,9 +13,15 @@ module UserSystem
       class UserSystemPlugin < Kanal::Core::Plugins::Plugin
         include Models
         include Helpers
+        include AutoCreator
```

Comment:
Using included module makes all methods of this module public in the extended class. Meaning user can write 

plugin.enable_telegram simply by mistake of autocomplete or something